droidlet/lowlevel/hello_robot/hello_robot_mover.py

```diff
@@ -110,26 +167,20 @@ def look_at(self, target, turn_base=True):
         logging.info(f"looking at x,y,z: {target}")
         
         pan_rad, tilt_rad = get_camera_angles(cam_pos, target)
+#        pan_res = angle_diff(pos[2], pan_rad)
         # For the Hello camera, negative tilt seems to be up, and positive tilt is down
         # For the locobot camera, it is the opposite
         # TODO: debug this further, and make things across robots consistent
-        tilt_rad = -tilt_rad
+        #tilt_rad = -tilt_rad
```

Comment:
nit: generally just delete code that you don't need instead of commenting them

Reply:
done